src/lib.rs

```diff
@@ -1,16 +1,35 @@
 use geo::{LineString, Point, Polygon};
+use std::default::Default;
+
+// See https://stackoverflow.com/questions/24047686
+#[derive(Debug)]
+pub struct Params {
+    n_circles: usize,
+    distances: Vec<f64>,
+}
+
+impl Default for Params {
+    fn default() -> Self {
+        // todo: distances should be:
+        // zonebuilder::zb_100_triangular_numbers
+        // 1    3    6   10   15   21   28   36   45   55   66 ...
+        Params { n_circles: 5, distances: 10}
```

Comment:
`distances` is a `Vec<f64>`, but here you've passed in a single f64. You can say `distances: vec![1.0, 3.0, 6.0, 10.0, 15.0]` instead. `vec![...]` is a macro that creates a vector and does `vector.push(x)` a bunch of times for you